app/controllers/SDESNotificationsController.scala

```diff
@@ -32,7 +32,9 @@ import scala.concurrent.ExecutionContext
 class SDESNotificationsController @Inject()(notificationCache: NotificationCache,
                                             authorisedRequest: AuthorisedRequest,
                                             dateTimeService: DateTimeService,
-                                            cc: ControllerComponents)(implicit ec: ExecutionContext) extends BackendController(cc) with ControllerChecks {
+                                            cc: ControllerComponents)
+                                           (implicit ec: ExecutionContext)
```

Comment:
I think the implicit can be moved to the end of the params here?

Reply:
Done